File: renderer/src/main/java/ca/on/hojat/renderer/svg/SVGParseException.java
package ca.on.hojat.renderer.svg;

/**
 * Thrown by the parser if a problem is found in the SVG file.
 */

public class SVGParseException extends org.xml.sax.SAXException {
    public SVGParseException(String msg) {
        super(msg);
    }

    public SVGParseException(String msg, Exception cause) {
        super(msg, cause);
    }
}

File: renderer/src/main/java/ca/on/hojat/renderer/svg/SVGParseException.kt
package ca.on.hojat.renderer.svg

import org.xml.sax.SAXException

/**
 * Thrown by the parser if a problem is found in the SVG file.
 */
class SVGParseException(msg: String, cause: Exception?) : SAXException(msg, cause) {

    constructor(msg: String) : this(msg, null)
}